composer.json

```diff
@@ -2,9 +2,9 @@
   "name": "vinaikopp/module-customer-password-command",
   "description": "Set a customer password with bin/magento",
   "require": {
-    "php": "^7.0.0|^7.1.0|^7.2.0",
-    "magento/module-customer": "^100.1.1|^101.0|^102.0",
-    "magento/framework": "^100.1.1|^101.0|^102.0",
+    "php": "^7.0.0|^7.1.0|^7.2.0|^7.3.0|^7.4.0",
+    "magento/module-customer": "^100.1.1|^101.0|^102.0|^103.0",
+    "magento/framework": "^100.1.1|^101.0|^102.0^103.0",
```

Comment:
Seems like a `||` is missing between `^102.0^103.0"`

Also, using a single pipe `|` is actually not right - it needs to be doubled: `||`.
This is obviously my mistake, but if you don't mind, could you please also fix that for the other dependencies?

Reply:
Agreed. Jpdated.